### DISCUSSION TITLE
Speed difference between RPi Zero W and RPi 2 Zero W

### DISCUSSION THREAD

**Discussion by hanshuebner** [tjboldt/Apple2-IO-RPi#86] [General]
Not much of a surprise, but I found the speed difference between a Raspberry Pi Zero W V1.1 and a Raspberry Pi Zero 2 quite amazing:

With the Pi Zero W, the Apple IIe boots to the ProDOS prompt in 84 seconds, while with the Pi Zero 2 only takes 23 seconds.

I had started with the Pi Zero W and found the boot times quite annoying, so I'm glad that I have a Pi Zero 2 now. :)

**Comment by tjboldt** [comment]
Have you built your image from the updated instructions a few days ago? It disables some unnecessary services and speeds up the boot a bit. The Zero W 2 is noticeably faster regardless. I was getting cold boot times around 45 seconds on the original W and about 18 seconds on the W 2. Warm boot is about two or three seconds.

**Reply by hanshuebner** [reply]
Yes, I've built the image from your recent instructions - I particularly appreciated the hint that the Imager can now set up the image at installation time!

It may well be that the SD card I am using is very slow, I'll try another one.  Overall, 20 seconds are bearable, but more than a minute is annoying to me.
